provider/resource_site_rule.go

```diff
@@ -63,14 +65,21 @@ func resourceSiteRule() *schema.Resource {
 							Description: "signal id to tag",
 							Optional:    true,
 						},
+						"response_code": {
+							Type:         schema.TypeInt,
+							Description:  "HTTP code agent for agent to respond with. range: 400-499, defaults to '406' if not provided",
+							Optional:     true,
+							ValidateFunc: validateActionResponseCode,
+							Default:      http.StatusNotAcceptable,
```

Comment:
with the other logic you added this might not be necessary anymore. can we set this back to nil?